src/main/resources/application.properties

```diff
@@ -0,0 +1,17 @@
+server.port=8080
+
+#Database
+spring.datasource.url=jdbc:postgresql://localhost:5432/tournament
+spring.datasource.username=postgres
+spring.datasource.password=root
+spring.datasource.driver-class-name=org.postgresql.Driver
+spring.jpa.database=postgresql
+spring.jpa.database-platform=org.hibernate.dialect.PostgreSQLDialect
+
+#Hibernate
+spring.jpa.hibernate.ddl-auto=none
+spring.jpa.show-sql=true
+spring.jpa.properties.hibernate.format_sql=true
+spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.PostgreSQL81Dialect
```

Comment:
Use latest PostgreSQL.
8.1 version is already dead.
https://www.postgresql.org/support/versioning/